rmw_connext_cpp/src/rmw_get_topic_info.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "rmw/get_topic_info.h"
+#include "rmw/topic_info_array.h"
+#include "rmw_connext_shared_cpp/topic_info.hpp"
+#include "rmw_connext_cpp/identifier.hpp"
+
+extern "C"
+{
+rmw_ret_t
+rmw_get_publishers_info_by_topic(
+  const rmw_node_t * node,
+  rcutils_allocator_t * allocator,
+  const char * topic_name,
+  bool no_mangle,
+  rmw_topic_info_array_t * publishers_info)
+{
+  return get_publishers_info_by_topic(rti_connext_identifier, node, allocator, topic_name,
+           no_mangle, publishers_info);
```

Comment:
nitpick: (not sure why the linter didn't catch this, or maybe it's an uncrustify bug that makes it have to be this) normally we only indent two spaces at a time, i.e. a multiple of two and only two from the previous ones, no "alignment" to the previous line

```c++
  return get_publishers_info_by_topic(rti_connext_identifier, node, allocator, topic_name,
    no_mangle, publishers_info);
```

Or

```c++
  return get_publishers_info_by_topic(
    rti_connext_identifier,
    node,
    allocator,
    topic_name,
    no_mangle,
    publishers_info);
```

Reply:
Went with the second option.